test/telepoison_test.exs

```diff
@@ -59,6 +59,20 @@ defmodule TelepoisonTest do
     assert {"app.callname", "mariorossi"} in elem(attributes, 4)
   end
 
+  test "resource route can be explicitly passed to Telepoison invocation" do
+    Telepoison.get!("http://localhost:8000/user/edit/24", [], resource_route: "/user/edit")
+
+    assert_receive {:span, span(attributes: attributes)}, 1000
+    assert {"http.route", "/user/edit"} in elem(attributes, 4)
```

Comment:
We may introduce some more standard attributes in Telepoison in the future, in which case this test may start failing not because the functionality is broken but just because the `http.route` attribute may not be number 4 anymore. Maybe it's worth rewriting it in a way that doesn't rely on the specific position? We only care about it's presence and value.

Reply:
I've updated some of the tests that rely on this - can look at the others if required, want to check if the update is okay first!